oktaawscli/okta_auth.py

```diff
@@ -142,6 +154,40 @@ def verify_single_factor(self, factor, state_token):
                         exit(1)
                     else:
                         time.sleep(0.5)
+
+            if factor['factorType'] == 'u2f':
+                devices = u2f.list_devices()
+                if len(devices) == 0:
+                    print("No U2F device found")
+                    exit(1)
+
+                challenge = dict()
+                challenge['appId'] = resp_json['_embedded']['factor']['profile']['appId']
+                challenge['version'] = resp_json['_embedded']['factor']['profile']['version']
+                challenge['keyHandle'] = resp_json['_embedded']['factor']['profile']['credentialId']
+                challenge['challenge'] = resp_json['_embedded']['factor']['_embedded']['challenge']['nonce']
+                auth_response = None
+                while not auth_response:
+                    for device in devices:
+                        with device as dev:
```

Comment:
Can we add some feedback to the user that they need to push the button on their U2F device?

Reply:
ah yeah, thanks for catching

Reply:
I can add this but here I would use `print()` as `self.logger` only shows WARNING or ERROR messages, which doesn't sound right.

Reply:
done (with `print()`)

Reply:
Fair point. I'd like to eventually reconcile all of the outputs and set the default log level to `INFO`, but more work is needed before that happens.